inference-engine/include/ie_blob.h

```diff
@@ -756,6 +824,30 @@ class TBlob : public MemoryBlob {
     void* getHandle() const noexcept override {
         return _handle;
     }
+
+    /**
+     * @brief Creates an new empty rvalue LockedMemory object.
+     *
+     * @return rvalue for the empty locked object of type T
+     */
+    virtual LockedMemory<T> data() const noexcept {
+        return std::move(lockme<T>());
```

Comment:
Return by value uses move as first option by default, do we need `std::move` here?

Reply:
this is the question for another PR

Reply:
Again, don't understand why. You don't change existing code here, right?

Reply:
this is `const` copy of non-const existing method